code/datums/setup_option/backgrounds/origin_akula.dm

```diff
@@ -55,4 +55,4 @@
 		STAT_BIO = 0,
 		STAT_MEC = 0,
 		STAT_COG = -5
-	)
\ No newline at end of file
+	)
```

Comment:
why?

Reply:
literally no idea, that wasn't even the dummy change I made (which I proceeded to revert because I was just using it to fork the codebase)